documentation/ame-guide/modules/ROOT/pages/namespace/export-package.adoc

```diff
@@ -4,7 +4,7 @@
 
 To export a namespace, proceed as follows:
 
-. Click *File* -> *Export Package* in the xref:getting-started/ui-overview.adoc#menu-file[file menu].
+. Open the xref:getting-started/ui-overview.adoc#menu-file[File menu] and select *Export Package*.
```

Comment:
Ah, ok also a quite large dark background for this overlay modal. maybe I was wrong with my previous comment. In any case -- let's have consistency in treating such modals in screenshots.

Reply:
![grafik](https://github.com/eclipse-esmf/esmf-aspect-model-editor/assets/101344507/2183c135-59f0-4b42-8462-143f77f81318)